x-pack/plugins/observability/common/utils/formatters/formatters.test.ts

```diff
@@ -5,9 +5,45 @@
  * 2.0.
  */
 
-import { asPercent, asDecimalOrInteger } from './formatters';
+import { asDecimal, asInteger, asPercent, asDecimalOrInteger } from './formatters';
 
 describe('formatters', () => {
+  describe('asDecimal', () => {
+    it('displays the not available label when the number is not finite', () => {
+      expect(asDecimal(Infinity)).toBe('N/A');
+      expect(asDecimal(-Infinity)).toBe('N/A');
+      expect(asDecimal(null)).toBe('N/A');
+      expect(asDecimal(undefined)).toBe('N/A');
+    });
+
+    it('displays the correct label when the number is finite', () => {
```

Comment:
Should we test negative numbers and numbers with more than one thousand separators (like (1,000,000,000)?